static/sass/styles.scss

```diff
@@ -778,14 +785,48 @@ $p-small-lh-diff: map-get($line-heights, default-text) -
   @include vf-icon-video-play($color-mid-dark);
 }
 
+// new homepage styles
+
+.is-graphite {
+  background: $colors--dark-theme--background-default;
+  color: $colors--dark-theme--text-default;
+}
+
+.is-graphite,
+.is-dark {
+  a:not([class^="p-button"]) {
+    color: #69c;
+
+    &:visited {
+      color: #ab83d3;
+    }
+  }
+}
+
+.p-heading--display {
+  font-size: 4rem;
+  font-style: normal;
+  font-weight: 100;
+  line-height: 4.5rem;
+  margin-bottom: 1.5rem;
+  margin-top: 0;
+  max-width: $text-max-width;
+  padding-top: 0;
+
+  @media (min-width: $breakpoint-heading-threshold) {
+    font-size: 5rem;
+    line-height: 5.5rem;
+  }
```

Comment:
This will be upstreammed later. Do you think it is worth delaying Marketing/PdM QA for this?

Reply:
Since we figured out a way to demo without having to merge this PR I think it'd be nice to at least have the values expressed in vanilla variables? I don't think anyone other than you or maybe bartek can work up the magic!